config/locales/en.yml

```diff
@@ -53,25 +73,56 @@ en:
+      for example DWP’s Universal Credit online, DVLA’s vehicle tax service, or HMRC’s
+      webchat. These services may set additional cookies and, if so, will have their
+      own cookie policy and banner linking to it.
+    always_on: They always need to be on.
+    clicks: what you click on while you're visiting the site
     confirmation_message: Government services may set additional cookies and, if so,
       will have their own cookie policy and banner.
     confirmation_previous_referrer_link: Go back to the page you were looking at
     confirmation_title: Your cookie settings were saved
     cookie_settings: Cookie settings
     cookies_on_govuk: Cookies on GOV.UK
+    essential_explanation: These essential cookies do things like remember your progress
+      through a form (for example a licence application)
+    explanation: Cookies are files saved on your phone, tablet or computer when you
+      visit a website.
+    find_out: Find out more about cookies on GOV.UK
+    four_types: We use 4 types of cookie. You can choose which cookies you're happy
+      for us to use.
+    google: We use Google Analytics to measure how you use the website so we can improve
+      it based on user needs. We do not allow Google to use or share the data about
+      how you use this site.
+    google_info: Google Analytics sets cookies that store anonymised information about
+    how_we_use: We use cookies to store information about how you use the GOV.UK website,
+      such as the pages you visit.
+    how_you_got: how you got to the site
+    javascript: 'We use Javascript to set most of our cookies. Unfortunately Javascript
+      is not running on your browser, so you cannot change your settings. You can
+      try:'
     off-campaigns: Do not use cookies that help with communications and marketing
     off-settings: Do not use cookies that remember my settings on the site
     off-usage: Do not use cookies that measure my website use
     on-campaigns: Use cookies that help with communications and marketing
     on-settings: Use cookies that remember my settings on the site
     on-usage: Use cookies that measure my website use
+    pages_visited: the pages you visit on GOV.UK and government digital services,
+      and how long you spend on each page
+    reload: reloading the page
     save_changes: Save changes
+    third_parties: These cookies may be set by third party websites and do things
+      like measure how you view YouTube videos that are on GOV.UK.
+    turn_on_javascript: turning on Javascript in your browser
     types:
       campaigns: Cookies that help with our communications and marketing
       essential: Strictly necessary cookies
       settings: Cookies that remember your settings
       usage: Cookies that measure website use
+  education: education
   electoral:
     registration:
       description: Need help? Get in touch with your local electoral registration
```

Comment:
`description` is in here twice?

Reply:
That's weird! I didn't add those but I'll check where they're being used and fix

Reply:
I checked and they're under different keys, so it should be fine.

One is

```
electoral:
  registration:
    description:
```

The other is 
```
electoral
  service:
    description:
```